doc/api/fs.md

```diff
@@ -725,19 +725,55 @@ added: v0.8.6
 -->
 
 * `fd` {Integer}
-* `len` {Integer}
+* `len` {Integer} default = `0`
 * `callback` {Function}
 
 Asynchronous ftruncate(2). No arguments other than a possible exception are
 given to the completion callback.
 
+If the file referred by the file descriptor has more number of bytes than the
+`len` then, only the first `len` bytes will be retained in the file.
+
+For example, the following program retains only the first four bytes of the file
+
+```
```

Comment:
Seems our documentation is sporadic at best about including `` `​`​`js `` in our documentation.

/cc @nodejs/documentation